dev/TeachingTip/TeachingTip.cpp

```diff
@@ -1375,20 +1379,49 @@ winrt::TeachingTipPlacementMode TeachingTip::DetermineEffectivePlacement()
 
 void TeachingTip::EstablishShadows()
 {
-#ifdef USE_INTERNAL_SDK
-    if (SharedHelpers::IsThemeShadowAvailable())
-    {
+#ifdef USE_INSIDER_SDK
+#ifdef BEAK_SHADOW
 #ifdef _DEBUG
-        // This facilitates an experiment around faking a proper beak shadow, shadows are expensive though so we don't want it present for release builds.
-        auto beakShadow = winrt::Windows::UI::Xaml::Media::ThemeShadow{};
-        beakShadow.Receivers().Append(m_target.get());
-        m_beakPolygon.get().Shadow(beakShadow);
-        m_beakPolygon.get().Translation({ m_beakPolygon.get().Translation().x, m_beakPolygon.get().Translation().y, m_beakElevation });
+    if (winrt::IUIElement10 beakPolygon_uiElement10 = m_contentRootGrid)
+    {
+        if (m_tipShadow)
+        {
+            if (!beakPolygon_uiElement10.Shadow())
+            {
+                // This facilitates an experiment around faking a proper beak shadow, shadows are expensive though so we don't want it present for release builds.
+                auto beakShadow = winrt::Windows::UI::Xaml::Media::ThemeShadow{};
+                beakShadow.Receivers().Append(m_target.get());
```

Comment:
Just curious, did this solve the "polygon being cast as rectangle" issue?

Reply:
I've removed the shadow from the polygon entirely because of this issue, instead of having it present on debug builds with elevation 0. So this isn't causing issues for the control, but is still a limitation of shadow.